lib/tasks/yaml_db_anonymizer.rake

```diff
@@ -1,11 +1,23 @@
+require 'zlib'
+
 namespace :db do
   namespace :data do
-    desc 'Dump anonymized database contents to db/data.yml'
-    task :dump_anonymized => :environment do
+    def anonymize_helper
       format_class = ENV['class'] || "YamlDb::Anonymizer::Helper"
       require format_class.split('::').first.underscore
       helper = format_class.constantize
-      SerializationHelper::Base.new(helper).dump File.join(Rails.root, 'db', "data.#{helper.extension}")
+    end
```

Comment:
This is quite [dangerous](http://techblog.thescore.com/2016/03/17/defined-methods-in-rake-tasks-you-re-gonna-have-a-bad-time/). Maybe just put this in a module, as described in the post?